NerdyNomads/Text-Savvy
# Add TextBoxPopUp Modal

## What does this PR do?
- Add TextBoxPopup Modal

### Screenshots (optional)
![Screenshot 2022-03-03 194759](https://user-images.githubusercontent.com/24909769/156683707-8899f00c-7acf-493f-ad7a-83d2033bb41d.png)
![Screenshot 2022-03-03 194911](https://user-images.githubusercontent.com/24909769/156683711-a51ecb10-8b4c-4627-a820-35f9a2fb13d5.png)

### Steps to test/review (optional)
- Run front-end
   - See if the component's size is responsive when resizing the window
   - Should add a scroll if the text overflows
   - should wrap to the text width if it is shorter than max width
   - open close should work
   - click to the link will redirect you to it at the moment
     - will ad copy to clipboard when clicked later 

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
